README.md

```diff
@@ -21,7 +28,15 @@ TODO:
 * Compiler
 * Webasm target
 
-## Code example:
+# Installation
+`UwUpp` can be installed on Arch-based systems with `yay uwupp-git`. For all other systems:
```

Comment:
Clarify that the package is on AUR, in case someone uses some other client instead of `yay`, or wants to install it manualy.

Reply:
thats a good point, lemmie edit that